scripts/linters/other_files_linter.py

```diff
@@ -342,6 +345,59 @@ def _check_that_workflow_steps_use_merge_action(
             for job in jobs_without_merge
         ]
 
+    def check_github_workflows_have_name(
+        self
+    ) -> concurrent_task_utils.TaskResult:
+        """Checks that all github actions workflow steps have a name.
+
+        Returns:
+            TaskResult. A TaskResult object describing any workflows
+            steps that do not have a name.
+        """
+        name = 'Github workflow steps have a name'
+        workflow_paths = {
+            os.path.join(WORKFLOWS_DIR, filename)
+            for filename in os.listdir(WORKFLOWS_DIR)
+            if re.search(WORKFLOW_FILENAME_REGEX, filename)
+        }
+        errors = []
+        for workflow_path in workflow_paths:
+            workflow_str = self.file_cache.read(workflow_path)
+            workflow_dict = yaml.load(workflow_str, Loader=yaml.Loader)
+            errors += self._check_that_workflow_steps_have_name(
+                workflow_dict, workflow_path)
+        return concurrent_task_utils.TaskResult(
+            name, bool(errors), errors, errors)
+
+    # Here we use type Any because the argument 'workflow_dict' accept
+    # dictionaries that represents the content of workflow YAML file and
+    # that dictionaries can contain various types of values.
```

Comment:
that dictionaries --> those dictionaries

Reply:
Done